### DISCUSSION TITLE
Add a option to Enable and Disable pen nib and pen button

### DISCUSSION THREAD

**Discussion by kckc977** [ppy/osu#17123] [Ideas] [answered]
is it possible to Enable and Disable pen nib in the tablet settings inside the game? instead of just disabling mouse button which disables everything including the pen nibs it could have a separate option to Enable and disable mouse buttons or pen nibs. i think it could be handy for most players.

**Comment by Theighlin** [comment]
If i'm understanding this right you're trying to have mouse buttons disabled and pen inputs enabled at the same time, and i assume it is during gameplay because the setting has no effect outside of it. Can you explain in what ways it would be "handy for most players"?

**Reply by kckc977** [reply]
well maybe some players like to make costume keypads that is easy and cheap to make using a broken mouse. and maybe be helpful to other players who like to use mouse in navigating beatmaps more and is a player that has a drag playstyle that dont want to use their pens to enable maybe??? well if its possible it would be great to know

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/issues/14037
